x/inflation/types/msg.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ sdk.Msg = &MsgUpdateParams{}
+
+// GetSigners returns the expected signers for a MsgUpdateParams message.
+func (m *MsgUpdateParams) GetSigners() []sdk.AccAddress {
+	addr, _ := sdk.AccAddressFromBech32(m.Authority)
```

Comment:
I see in other modules that the `MustAccAddressFromBech32` method is used on the `GetSigners()` instead of `AccAddressFromBech32`. Is there a particular reason why we are using this other method in this case?

```suggestion
	addr := sdk.MustAccAddressFromBech32(m.Authority)
```